### DISCUSSION TITLE
Downloading data.zip on https://ftp.tugraz.at/outgoing/ITSG/groops/ is too slow

### DISCUSSION THREAD

**Discussion by whcyoucan** [groops-devs/groops#129] [Q&A] [answered]
On https://ftp.tugraz.at/outgoing/ITSG/groops/, you must download a compressed file with a size of 10 G（data.zip）. Is there any way to download only the files in one directory?

**Comment by akvas** [comment, marked as answer]
Hi @whcyoucan, 

the [data repository](https://ftp.tugraz.at/outgoing/ITSG/groops/) is an ordinary [FTP](https://en.wikipedia.org/wiki/File_Transfer_Protocol) server, from which you can download individual files and directories.

Cheers,
Andreas

**Reply by whcyoucan** [reply]
It seems that the xml file cannot be downloaded directly, only by other means, and it seems that the whole directory cannot be downloaded?

**Comment by Waley04** [comment]
Hi @whcyoucan,

Alternatively, you may also download similar GNSS data from NASA's CDDIS archive in the repro3 subdirectories. For example, you may check [here](https://cddis.nasa.gov/archive/gnss/products/2050/repro3/).

Best wishes,

Waley.

**Reply by whcyoucan** [reply]
OK，thanks.

**Comment by tmayerguerr** [comment]
Dear all,
For clarification: The data are provided twice at our FTP. As bunch of single files in the data/ directory and the complete directory as one single zip file. The content should be the same.
regards
Torsten
